### DISCUSSION TITLE
Welcome to SharpMap Discussions!

### DISCUSSION THREAD

**Discussion by FObermaier** [SharpMap/SharpMap#202] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by HikeAndMap** [comment]
I've downloaded this through the git clone option in visual studio and it errors.
Tons of errors: 
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB4024	The imported project file "C:\Users\vince\.nuget\packages\gitversiontask\5.5.1\build\GitVersionTask.targets" could not be loaded. Root element is missing.	C:\Users\vince\Source\Repos\SharpMap\SharpMap.sln	C:\Users\vince\Source\Repos\SharpMap\Examples\DemoWinForm\obj\DemoWinForm.csproj.nuget.g.targets	10
```

although I have the NuGet and everything installed.

Then I thought: "okay, going to follow the tutorial - slim installation, no nuget no git.. just the minimum I need"
https://github.com/SharpMap/SharpMap/wiki 
started with the wiki as "tutorial"
works.. until I come here:
https://github.com/SharpMap/SharpMap/wiki/Adding-a-Tiled-layer-as-Background

From there on it doesn't work anymore

```
ProjNet.CoordinateSystems.Transformations.CoordinateTransformationFactory ctFact = new ProjNet.CoordinateSystems.Transformations.CoordinateTransformationFactory();
            vlay.CoordinateTransformation = ctFact.CreateFromCoordinateSystems(ProjNet.CoordinateSystems.GeographicCoordinateSystem.WGS84, ProjNet.CoordinateSystems.ProjectedCoordinateSystem.WebMercator);
            vlay.ReverseCoordinateTransformation = ctFact.CreateFromCoordinateSystems(ProjNet.CoordinateSystems.ProjectedCoordinateSystem.WebMercator, ProjNet.CoordinateSystems.GeographicCoordinateSystem.WGS84);

    mapBox1.Map.BackgroundLayer.Add(new SharpMap.Layers.TileAsyncLayer(
                new BruTile.Web.OsmTileSource(), "OSM"));
```
                
the transformation give errors on casting 
                
the tileAsyncLayer doesn't exist..
eventually I got that running though..

but again, the transformation thingy keeps annoying me with the message about casting 
                
                
is there a tutorial that actually works?
or a https://github.com/SharpMap/SharpMap.git that actually works out of the box?
